### DISCUSSION TITLE
de/2-03-kata-2/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#48] [General]
# de/2-03-kata-2/

https://ai.lernos.org/de/2-03-kata-2/

**Comment by felixharling** [comment]
Im Leitfaden wird "Unternehmen" genutzt. Das schliesst Nutzer*innen aus, die aus der Verwaltung, Vereinen oder anderen Organisationen kommen. Daher bitte "Organisation" als allgemeineren Begriff durchgehend nutzen.

**Reply by simondueckert** [reply]
Das stimmt, wir gehen den Leitfaden im Nachgang zum MOOC nochmal komplett durch und entfernen alle Einschränkungen auf "Unternehmen". Ist als Issue #60 aufgenommen.

**Reply by maki-amw** [reply]
Das wird hier in der Kata nun direkt entsprechend berücksichtigt und angepasst.

**Comment by MartinaQuiterio** [comment]
Bitte Links zu anderen Seiten & Tools in einem separaten Fenster öffnen lassen - das erleichtert die Navigation ;-)

**Reply by simondueckert** [reply]
Das geht in der von uns verwendeten Sprache Markdown nicht und wir wollen die Entscheidung, wo der Link aufgeht, auch den Nutzenden überlassen. Wenn du beim Klick auf einen Link die Strg-Taste drückst, geht der Link im neuen Fenster auf.

**Reply by olelai** [reply]
Für Mac-Nutzer:innen: die Cmd-Taste

**Reply by rednaxelaesor** [reply]
Hallo Martina, mir geht es genau so. Schade das es technisch nicht geht. Das "Entscheidungsargument" von Simon teile ich nicht vollumfänglich. Wäre es umgekehrt (Neuer Tab als Default) , würde die Argumentation genau so funktionieren.  Weiterer Tipp: Auch mit der rechten Maustaste kannst du den Link in einem neuen Tab öffnen. (Windows)

**Comment by simondueckert** [comment]
Die Kata 2 für den Hashtag#kimooc24 ist wohl eher für Text-KIs gedacht ... das Bild bekomme ich von meiner Bild-KI Hashtag#StableDiffusion 😂 Ich werde die Aufgabe etwas modifizieren und versuchen, den Time-for-Change-Frosch nachzuprompten: https://www.linkedin.com/feed/update/urn:li:activity:7194044195622297602/

**Reply by olelai** [reply]
Der gesamte Leitfaden konzentriert sich auf Text-KIs und lässt Bild-KIs außen vor. Vielleicht wäre zu Bild-KIs ein eigener Leitfaden sinnvoll, weil Diffusion-Modelle anders funktionieren und manche der Katas für Bild-KIs sinnlos sind. Andere wiederum kann  man gut anwenden, z.B. Kata 5.

**Reply by maki-amw** [reply]
@simondueckert vielleicht können wir hier direkt eine modifizierte Variante für Bild-KIs auf Basis Deines damaligen Versuchs ergänzen?

**Comment by rednaxelaesor** [comment]
Typo: Aufgabe: Wähle eine~n~ KI-Tool oder probiere mehrere aus

**Reply by maki-amw** [reply]
Done. Vielen Dank für den Hinweis!

**Comment by rednaxelaesor** [comment]
Verweis auf Kata 5 "Dies ist vor allem für die nachfolgenden Aufgaben wichtig und wird später in [Kata 5] nochmals aufgegriffen und vertieft." ist ohne Link-Funktion

**Reply by maki-amw** [reply]
Done. Vielen Dank für den Hinweis!
